src/System.Net.Security/src/Resources/Strings.resx

```diff
@@ -417,6 +417,9 @@
   <data name="net_alpn_config_failed" xml:space="preserve">
     <value>ALPN configuration failed on this platform.</value>
   </data>
+  <data name="net_alpn_failed" xml:space="preserve">
+    <value>No common application protocol exists between the client and the server. Application negotiation failed.</value>
```

Comment:
Is this copied from somewhere?  If not I'd change it to first state that the negotation failed, then say why, e.g.
```
Application negotiation failed due to lack of a common application protocol between the client and the server.
```

Reply:
This exception message is given by schannel on windows, just copied that message.

Reply:
Ok